packages/react-composites/src/composites/CallComposite/ScreenSharePopup.tsx

```diff
@@ -87,7 +87,9 @@ export const ScreenSharePopup = (props: ScreenSharePopupProps): JSX.Element => {
             disabled={stoppingInProgress}
           />
         </Stack>
-        <FocusTrapCallout focusTrapProps={{ isClickableOutsideFocusTrap: true }}></FocusTrapCallout>
+        <FocusTrapCallout
+          focusTrapProps={{ isClickableOutsideFocusTrap: true, forceFocusInsideTrap: false }}
```

Comment:
This workaround is causing problem when we get the right behavior of re-rendering(which is making this workaround behaves correctly), so an additional parameter is needed to solve the bug